pom.xml

```diff
@@ -19,6 +19,7 @@
   <properties>
     <java.version>1.7</java.version>
     <avro.version>1.7.7</avro.version>
+    <scala.version>2.10</scala.version>
```

Comment:
Could you document this in here? It's not immediately obvious that this is used by `About.java`, and I'm worried that without docs, this will get removed by someone at a later date.